samples/bluetooth/peripheral_fast_pair/prj.conf

```diff
@@ -18,6 +18,8 @@ CONFIG_NVS=y
 CONFIG_SETTINGS_NVS=y
 CONFIG_SETTINGS=y
 CONFIG_BT_SETTINGS=y
+# Disable CCC Lazy Loading to workaround rebond issue related to NCSDK-22921 Zephyr bug.
```

Comment:
Can we move it next to `CONFIG_BT_AUTO_PHY_UPDATE=n` (to keep the workarounds together)?

Reply:
Can we explain it a bit more (also in release notes)? A few sentences would be great since a lot of readers cannot access our Jira to get more details.